test/integration/client-side-encryption/driver.test.ts

```diff
@@ -1,20 +1,32 @@
 import { EJSON, UUID } from 'bson';
 import { expect } from 'chai';
 import * as crypto from 'crypto';
+import * as sinon from 'sinon';
 
 // eslint-disable-next-line @typescript-eslint/no-restricted-imports
 import { ClientEncryption } from '../../../src/client-side-encryption/client_encryption';
-import { type Collection, type CommandStartedEvent, type MongoClient } from '../../mongodb';
-import * as BSON from '../../mongodb';
-import { getEncryptExtraOptions } from '../../tools/utils';
-
-const metadata = {
+import {
+  BSON,
+  type Collection,
+  type CommandStartedEvent,
+  type MongoClient,
+  MongoOperationTimeoutError,
+  StateMachine
+} from '../../mongodb';
+import { type FailPoint, getEncryptExtraOptions } from '../../tools/utils';
+
+const metadata: MongoDBMetadataUI = {
   requires: {
     mongodb: '>=4.2.0',
     clientSideEncryption: true
   }
 };
 
+const LOCAL_KEY = Buffer.from(
```

Comment:
This is only used for CSOT tests.  Can we move this key to the appropriate describe block?